pkg/cover/html.go

```diff
@@ -211,67 +211,44 @@ func (rg *ReportGenerator) DoRawCoverFiles(w io.Writer, params CoverHandlerParam
 	return nil
 }
 
-type covCallbacksStat struct {
-	TotalCount   int `json:"total_cb_count"`
-	CoveredCount int `json:"covered_cb_count"`
-}
-
-type covFuncDetails struct {
-	covCallbacksStat
-}
-
-type covFileDetails struct {
-	covCallbacksStat
-	Functions map[string]*covFuncDetails `json:"functions,omitempty"`
-}
-
 type coverageInfo struct {
-	Version int `json:"version"`
-	covCallbacksStat
-	Files map[string]*covFileDetails `json:"files,omitempty"`
+	Version   int    `json:"version"`
+	FilePath  string `json:"file_path"`
+	FuncName  string `json:"func_name"`
+	StartLine int    `json:"sl"`
+	StartCol  int    `json:"sc"`
+	EndLine   int    `json:"el"`
+	EndCol    int    `json:"ec"`
+	HitCount  int    `json:"cb_hit_count"`
```

Comment:
Should we be looking at this PR right now, or it's some temp? This again has "cb_hit_count".

Reply:
No no no. Please don't.
I'll rebase it on the landed #4577, fix and then ask for the review.